src/redux/reducers/settings.js

```diff
@@ -33,89 +30,83 @@ const defaultState = {
   serverUrl: 'https://api.susi.ai',
   backgroundImage: '',
   messageBackgroundImage: '',
+  avatarType: 'gravatar',
 };
 
 export default handleActions(
   {
-    [actionTypes.APP_GET_USER_SETTINGS](state, { payload }) {
-      return {
-        ...payload,
-      };
-    },
-    [actionTypes.APP_SET_USER_SETTINGS](state, { payload }) {
-      return {
-        ...state,
-      };
-    },
-    [actionTypes.SETTINGS_REMOVE_USER_DEVICE](state, { payload }) {
-      return {
-        ...state,
-      };
-    },
-    [actionTypes.SETTINGS_ADD_USER_DEVICE](state, { payload }) {
-      return {
-        ...state,
-      };
-    },
-
-    [actionTypes.SETTINGS_SET_ACCOUNT_SETTINGS](state, { payload }) {
-      const { userName, timeZone, prefLanguage } = payload;
-      return {
-        ...state,
+    [actionTypes.SETTINGS_GET_USER_SETTINGS](state, { payload }) {
+      const { settings } = payload;
+      const {
+        theme,
+        server,
+        serverUrl,
+        enterAsSend,
+        micInput,
+        speechOutput,
+        speechOutputAlways,
+        speechRate,
+        speechPitch,
+        ttsLanguage,
         userName,
-        timeZone,
         prefLanguage,
-      };
-    },
-    [actionTypes.SETTINGS_SET_MOBILE_SETTINGS](state, { payload }) {
-      const { phoneNo, countryCode, countryDialCode } = payload;
-      return {
-        ...state,
-        phoneNo,
+        timeZone,
         countryCode,
         countryDialCode,
-      };
-    },
-    [actionTypes.SETTINGS_SET_CHAT_PREFERENCES_SETTINGS](state, { payload }) {
-      const { enterAsSend } = payload;
+        phoneNo,
+        checked,
+        backgroundImage,
+        messageBackgroundImage,
+        avatarType,
+      } = settings;
+      let { customThemeValue } = settings;
+      const themeArray = customThemeValue.split(',').map(value => `#${value}`);
```

Comment:
What if the use uses light or dark theme instead of custom?

Reply:
@akshatnitd If we use light or dark theme, the values will be fixed for both of them.